schematic_api/api/routes.py

```diff
@@ -170,7 +168,19 @@ def parse_bool(str_bool):
         raise ValueError(
             "String boolean does not appear to be true or false. Please verify input."
         )
-        
+
+def return_as_json(manifest_local_file_path):
+    manifest_csv = pd.read_csv(manifest_local_file_path)
+    try:
+        manifest_json = manifest_csv.to_dict(orient="records")
+        return manifest_json
+    except JSONDecodeError as e:
+        logger.error("Json data is not formatted correctly", e)
```

Comment:
Is this logging the original error and your message to whatever logger was initialized before?
It might be more helpful to [wrap the caught errors](https://peps.python.org/pep-3134/#explicit-exception-chaining) with the useful messages you've added so they're included in the stack trace instead of logging them to the logger where they might be missed, like how you've done in other parts of the code. Like:
```
except JSONDecodeError as e:
    SomeSpecificFormatError("Json data is not formatted correctly") from e
```
Apologizes if I've misunderstood the output of the log and it already does something similar!

Reply:
Yeah I think here maybe I don't even need the try and except block. The error message that I wrote didn't really add much values

Reply:
Let me do a light refactor of routes.py later this sprint and try clearing up the code.